docker-compose.custom-test.yml

```
@@ -15,24 +15,47 @@ services:
   skosmos-dev:
     container_name: evokstest_skosmos-dev
-    restart: no
+    restart: "no"
 
   skosmos-live:
     container_name: evokstest_skosmos-live
-    restart: no
+    restart: "no"
+
+
+services:
```

yamllint

[error] duplication of key "services" in mapping

(key-duplicates)